lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -217,13 +218,19 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
   if (auto Sym = getInjectedSymbols(Name))
     return Sym;
 
+  const std::string SymName = Name;
```

Comment:
Not a bad idea.
Averaging the time to run each test in the suite 6 times before and after showed a 7% drop in execution time. Could be noise but it's definitely more resource friendly this way.